### ISSUE TITLE
Not an issue, but a thank you

### ISSUE THREAD

**Issue by geekgirl101** [Moop-HorizonXI/FFXI-HD-hunting-maps#2]
Hi, I wanted to thank you for the work you've done so far in editing and remaking some of the HD maps for FFXI.  I know you've not worked on them in a while especially since the people at Remapster have taken a long break themselves from redrawing many of the missing maps in the game.  I've tried to redraw a couple of the missing ones myself even though I don't have half the tools or knowledge, and yeah it's a lot of effort just to draw a single map.  I appreciate everything you've done so far and just wanted to let you know that I'm very greatful for what you've done to help with the HorizonXI community in providing better maps and hope you'll some day find some time to continue with the other maps.  I've told several of my friends who use modded maps where to find yours for far better versions and they're all absolutely thankful to have something that isn't a blurry and unreadable mess.

**Comment by Moop-HorizonXI** [comment]
that is very kind of you and I am happy to hear you and your friends are enjoying them, thank you!

The work and accuracy of information they put into the remapster maps is something I didnt want to dismiss when working on additional maps. Sadly I've just not been able to invest in the time to level high enough to survey zones nor have any friends who play so it's tough to keep motivated

I will pop on to level my beastmaster here and there (level 35 at the time of writing this). This was my way of planning to hit 75 as a BST before continuing because I figured it would be a good way for me to go solo and dive into each zone for information to include on any new maps

I may do more at some point in the future and have always wanted to update all maps to reflect information that only pertains to horizon-xi and not retail.

**Comment by geekgirl101** [comment]
Yeah I understand and appreciate the amount of effort involved.  I've had a dabble with a few of them with me having plenty of free time on my hands.  Some seem fairly easy like the Ifrit's Cauldron maps, but then you got ones like Ro'mave that took me a few weeks redrawing it so the lines didn't look janky.  I don't have the same tools and knowledge that the guys at Remapster have to draw their maps based on the actual terrain, but I've been cross-referencing between the official maps and the ones on the wiki and visiting the zones to make sure they're as close as possible.  It's still very time consuming.

**Comment by ZellithS** [comment]
I also want to say thank you.  These maps are great!  If you ever want to pt or something in game, shoot me a tell! Though Im a bit of a scrub who only just beat Maat for LB5!
